### ISSUE TITLE
Implement toString on JwtMap

### ISSUE THREAD

**Issue by jahlborn** [jwtk/jjwt#58]
I'm playing with the library right now, and most of the default class impls don't have useful toString() implementations on them.  JwtMap could have a simple toString() (delegate to the internal map).  Would be nice to add toString() to some of the other value types (e.g. DefaultJws, etc.).

**Comment by lhazlewood** [comment]
Pull requests welcome!

**Comment by jahlborn** [comment]
```
@Override
public String toString() {
  return map.toString();
}
```

**Comment by lhazlewood** [comment]
I see what you're saying now - thanks!

**Comment by lhazlewood** [comment]
FWIW, this was more work than just what was suggested - both the DefaultJws and DefaultJwt classes needed to be modified and test cases provided for both to retain 100% test coverage.  Please submit a Pull Request with tests next time.

**Comment by lhazlewood** [comment]
Implemented in https://github.com/jwtk/jjwt/commit/4d230a07256401a170911786de180886db28a003.  Will be released in 0.6.
